api/src/test/java/org/openmrs/module/fhir/api/ObsServiceTest.java

```diff
@@ -118,4 +120,16 @@ public void searchObsByPerson_shouldReturnMatchingObservationList() {
 		assertNotNull(obs);
 		assertEquals(3, obs.size());
 	}
+	
+	@Test
+	public void deleteObs_shouldDeleteTheSpecifiedObs()
+	{
+		org.openmrs.api.ObsService obsService= Context.getObsService();
```

Comment:
make sure things are formatter correctly using openmrs code formatter. Need to have spaces from left and right of a space. I'll correct them and merge for now.